### ISSUE TITLE
Please provide Configuration files and firmware source code.

### ISSUE THREAD

**Issue by ellensp** [Creality3DPrinting/CR10-V2#3]
All documentation is missing.

No Firmware
No Source
No Config files
not even a user guide. 

Please provide all of these.

**Comment by aleohl** [comment]
Bump this issue.

Really @Creality3DPrinting? Didn't you learn anything from last time?

**Comment by MachineryEnchantress** [comment]
Source code here: https://www.creality.com/download/firmware
It's a pain to get them to use Github, I'll move it over later.

**Comment by aleohl** [comment]
@sexycyborg do you know if they can provide PCB, Cables etc.? 
I need to fix the connector for the bed. I rather have a pinout than trial-and-error.

**Comment by aleartdev** [comment]
+1 @aleohl. I also need pinouts

**Comment by acguardia** [comment]
No Schematics.
No Pins Maps to the board.
No PCB Layout.
![creality-cr-10-v2-motherboard-v252](https://user-images.githubusercontent.com/10907735/115657329-0d6bc100-a30d-11eb-9e52-499c415f2df7.jpg)

**Comment by aleohl** [comment]
Seems like it's not open source, check #4

**Comment by ellensp** [comment]
Even if the controller and machine is not open source, they are using Marlin, and as such are required to publish the firmware source including Config files as specified in the Marlin license agreement.
But they wont because Creality is a disrespectful company that doesn't care about licencing.

**Comment by stweed69** [comment]
did anyone ever find or come up with pins pdf...

**Comment by karolprzyb** [comment]
Does anyone have the source provided with their firmware updates? Looks like the links are dead and the v2 is no longer in the download section on the website. This is very irritating. Was hoping to at least get the source so I wouldn't have to probe the whole board.

For anyone looking for configurations you can grab them from Marlin here:
https://github.com/MarlinFirmware/Configurations/tree/bugfix-2.1.x/config/examples/Creality

This should be enough to start to figure out whats going on.

**Comment by ellensp** [comment]
Don't know why you would want the really old version of Marlin from Creality, When marlin has modern configs for it but...

But here is a copy (originally a rar, but github doesn't want to allow that) 

[CR-10V2 Source Code.zip](https://github.com/Creality3DPrinting/CR10-V2/files/11000077/CR-10V2.Source.Code.zip)
Annother version with bltouch can be found here https://drive.google.com/drive/folders/1N_RVXtakOYS0uQJso0YY8yp2j1aFmoik

**Comment by karolprzyb** [comment]
Yup, found it later the same day. Thanks! Did not realize they had configs in the repo. Was hoping maybe the source creality would have would be a little more clear on pinning back to the board outputs in comments or something related.

**Comment by karolprzyb** [comment]
In case anyone finds it useful here are the pin-outs for EXP1 and EXP2.

EDIT: Matches ramps in marlin configs after looking at it a bit more.

I did not look at too much else yet.

![CR10v2_pinout_2_5_2](https://user-images.githubusercontent.com/1251255/226389810-77f35efd-ab14-4020-94cf-64c753cab21d.png)
